providers/hubspot/record.go

```diff
@@ -0,0 +1,55 @@
+package hubspot
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+
+	"github.com/amp-labs/connectors/common"
+)
+
+/*
+   docs:
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/companies
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/contacts
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/deals
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/tickets
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/line_items
+   https://developers.hubspot.com/beta-docs/reference/api/crm/objects/products
+*/
+
+//nolint:gochecknoglobals
+var getRecordSupportedObjectsToPathMap = map[string]string{
+	"company":   "companies",
+	"contact":   "contacts",
+	"deal":      "deals",
+	"ticket":    "tickets",
+	"line_item": "line_items",
+	"product":   "products",
+}
+
+var ErrUnsupportedObject = errors.New("unsupported object")
+
+func (c *Connector) GetRecord(ctx context.Context, objectName string, recordId string) (*common.ReadResultRow, error) {
+	objectNameInPath, ok := getRecordSupportedObjectsToPathMap[objectName]
+	if !ok {
+		return nil, fmt.Errorf("%w: %s", ErrUnsupportedObject, objectName)
```

Comment:
I am not sure if the common error fits in here:
```
common.ErrOperationNotSupportedForObject
```

Reply:
actually I looked into the error, and the error test wasn't clearly fitting. GetRecord is not much tied to `operation`. But I changed the text.

Reply:
Yeah the error message is a bit distant. But if we are planning to have `GetRecord` it would be nice to have an error in `common` package to use the same language among connectors